test/index.js

```diff
@@ -34,17 +36,121 @@ describe('Wigzo', function () {
   describe('Constructing the integration', function () {
     it('should have the right settings', function () {
       analytics.compare(Wigzo, Integration('Wigzo')
+        .readyOnInitialize()
+        .global('wigzo')
+        .option('licenseCode', ''));
     })
-  })
 
-  describe('#initialize', function () {
-    it('should call this.ready() with valid options', function () {
-      analytics.stub(wigzo, 'ready')
-      wigzo.initialize()
-      analytics.called(wigzo.ready)
-    })
+    describe('#initialize', function () {
+      it('should call this.ready() with valid options', function () {
+        analytics.stub(wigzo, 'ready')
+        wigzo.initialize()
+        analytics.called(wigzo.ready)
+      })
+
+      it('should create window.wigzo', function() {
```

Comment:
i think these should all fail because you're not calling `wigzo.initialize()` before each of these assertions.